### PR TITLE
Update artifact configuration page

### PR DESCRIPTION
**Pull request by guilgaly**

### CODE REVIEW

File: content/reference/user/artifact_conf/index.md [hunk 1]
```diff
@@ -43,13 +40,14 @@ In order to package a bundle of your simulation, refer to the [Developer documen
 
 {{< img src="artifact-upload-empty.png" alt="Artifact upload empty" >}}
 
-You can then drag and drop your artifact to filled the form and proceed to upload.
+You can then drag and drop your artifact to fill the form and click on the **Upload** button.
 
 {{< img src="artifact-upload-filled.png" alt="Artifact upload filled" >}}
 
-You can upload multiple artifact concurrently, progress can be tracked from anywhere in the application through **Uploads** in the navbar.
+You can upload multiple artifact concurrently. Upload progress will be shown in the Artifacts table.
```
Comment (Isammoc) [inline content/reference/user/artifact_conf/index.md:47]:
```suggestion
You can upload multiple artifacts concurrently. Upload progress will be shown in the Artifacts table.
```

Comment (Isammoc) [reply content/reference/user/artifact_conf/index.md:47]: Active voice
```suggestion
You can upload multiple artifact concurrently. The Artifacts table will show the upload progress.
```

File: content/reference/user/artifact_conf/index.md [hunk 2]
```diff
@@ -43,13 +40,14 @@ In order to package a bundle of your simulation, refer to the [Developer documen
...
+
+An upload in progress can be cancelled by clicking on the cancel icon
```
Comment (Isammoc) [inline content/reference/user/artifact_conf/index.md:49]: Active voice
```suggestion
You can cancel an upload in progress by clicking on the cancel icon.
```

**Review by Ven0mX — APPROVED** [review]

File: content/reference/user/artifact_conf/index.md [hunk 3]
```diff
@@ -43,13 +40,14 @@ In order to package a bundle of your simulation, refer to the [Developer documen
 
 {{< img src="artifact-upload-empty.png" alt="Artifact upload empty" >}}
 
-You can then drag and drop your artifact to filled the form and proceed to upload.
```
Comment (ccedric) [inline content/reference/user/artifact_conf/index.md:46]: on line 38, can you modify: [Artifact Generation documentation]()

File: content/reference/user/artifact_conf/index.md [hunk 4]
```diff
@@ -43,13 +40,14 @@ In order to package a bundle of your simulation, refer to the [Developer documen
...
+You can then drag and drop your artifact to fill the form and click on the **Upload** button.
 
 {{< img src="artifact-upload-filled.png" alt="Artifact upload filled" >}}
 
-You can upload multiple artifact concurrently, progress can be tracked from anywhere in the application through **Uploads** in the navbar.
+You can upload multiple artifacts concurrently. Upload progress will be shown in the Artifacts table.
```
Comment (ccedric) [inline content/reference/user/artifact_conf/index.md:47]: Upload**s** progress

File: content/reference/user/artifact_conf/index.md [hunk 5]
```diff
@@ -58,7 +56,7 @@ You can also upload artifacts programmatically with our REST API.
 
 You'll need:
 * an [API token]({{< ref "../../admin/api_tokens" >}}) with at least the `Artifacts` permission
-* the Artifact's ID you can copy from the WebUI.
+* the Artifact's ID, which you can copy from the WebUI.
```
Comment (ccedric) [inline content/reference/user/artifact_conf/index.md:59]: , which can be copied from the artifacts table.

Comment (Isammoc) [reply content/reference/user/artifact_conf/index.md:59]: Always prefer the active voice!

**Review by Isammoc — APPROVED** [review]

File: content/reference/user/artifact_conf/index.md [hunk 6]
```diff
@@ -11,25 +11,22 @@ weight: 10050
 
 To access the Artifacts section, click on **Artifacts** in the navbar.
```
Comment (ccedric) [inline content/reference/user/artifact_conf/index.md:12]: in the SideNav, not the navbar

Comment (guilgaly) [reply content/reference/user/artifact_conf/index.md:12]: Isn't the side bar called the "navigation bar" elsewhere in the docs? (Either way, neither "SideNav" nor "navbar" seem appropriate terminology for the user facing docs.)

Comment (ccedric) [reply content/reference/user/artifact_conf/index.md:12]: Sidenav is the one on the left side, and navbar at the top side. Indeed, the naming can get a bit confusing, and may need a change

Comment (guilgaly) [reply content/reference/user/artifact_conf/index.md:12]: I checked, we do seem to call it navigation bar everywhere else in the docs. I'm going to use that; if it's not correct then it should be changed globally.

**Pull request merged**